hilti/runtime/include/global-state.h

```diff
@@ -69,6 +70,12 @@ struct GlobalState {
      * still need this information.
      */
     std::vector<hilti::rt::detail::HiltiModule> hilti_modules;
+
+    /** The main coroutine for fiber execution. */
+    std::optional<::Fiber> main_co;
+
+    /** The currently executing fiber .*/
+    ::Fiber* cur_co = nullptr;
```

Comment:
I''m not quite seeing yet what we need `cur_co` for? Don't we always know the two fibers that we need to switch between (`main` && current fiber inside our ``Fiber`` class.

If we need it, it should also move into `Context`.

Reply:
Ah, of course! Nevermind.